### ISSUE TITLE
`obs_weight` should be called `obsweight`

### ISSUE THREAD

**Issue by lassepe** [JuliaPOMDP/POMDPModelTools.jl#29]
This is only a cosmetic change but I feel like `obs_weight` should rather be called `obsweight` at some point. It would be more consistent with our general naming pattern (e.g. `initialobs` instead of `initial_obs` in `POMDPs.jl`).

**Comment by zsunberg** [comment]
Yeah, we should probably review all the names in POMDPModelTools and upgrade them at once
